media-sound/whipper/whipper-0.7.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python CD-DA ripper preferring accuracy over speed (forked from morituri)"
+HOMEPAGE="https://github.com/JoeLametta/whipper"
+SRC_URI="https://github.com/JoeLametta/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+CDEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	media-libs/libsndfile
+"
+RDEPEND="
+	${PYTHON_DEPS}
+	${CDEPEND}
+	app-cdr/cdrdao
+	>=dev-libs/libcdio-paranoia-0.94_p2
+	dev-python/cddb-py[${PYTHON_USEDEP}]
+	dev-python/pycdio[${PYTHON_USEDEP}]
+	dev-python/pygobject:3=[${PYTHON_USEDEP}]
+	dev-python/python-musicbrainz-ngs[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	media-libs/flac
+	media-libs/mutagen[${PYTHON_USEDEP}]
+	media-sound/sox[flac]
+"
+DEPEND="${CDEPEND}"
+
+PATCHES=( "${FILESDIR}/${P}-cdparanoia-name-fix.patch" )
+
+src_prepare() {
+	distutils-r1_src_prepare
+}
+
+src_compile() {
+	distutils-r1_src_compile
+	emake -C src
+}
+
+src_test() {
+	distutils-r1_src_test
```

Comment:
Equal to the defualt, and doesn't do anything unless you declare `python_test()`. Read https://wiki.gentoo.org/wiki/Project:Python/Tests and https://wiki.gentoo.org/wiki/Project:Python/distutils-r1#Phase_functions

Reply:
I tried to do tests with
```
python_test() {
	"${EPYTHON}" -m unittest discover -v
}
```
like is [travis doing on upstream git repository](https://github.com/JoeLametta/whipper/blob/master/.travis.yml#L28-L30)

This gives this result in [logs](https://ptpb.pw/CZn3) actually.

Reply:
Which means you're missing test-dependencies. Add them conditionally to `test?`.

Reply:
Apparently the missing dep is `dev-python/twisted` a networking framework.
I should preferably disable/remove these tests.